config.example.yaml

```diff
@@ -146,6 +146,7 @@ services:
     conf: 
       port: 7231
       spec: *spec
+      saltKey: secret
```

Comment:
`saltKey` is easy to misunderstand as something connected to salt, the config management system. Perhaps call it just 'salt'?